lldb/include/lldb/Core/Progress.h

```diff
@@ -117,6 +127,7 @@ class Progress {
   /// to ensure that we don't send progress updates after progress has
   /// completed.
   bool m_complete = false;
+  bool m_type;
```

Comment:
This is confusing, why would you store your enum value in a `bool`

Reply:
Holdover from when I used a bool for this value before switching an enum when I put the patch up.